membership-attribute-service/app/models/AccountDetails.scala

```diff
@@ -8,13 +8,21 @@ import play.api.mvc.Results.Ok
 
 object AccountDetails {
 
-  implicit class ResultLike(details: (Contact, PaymentDetails, String)) {
+  implicit class ResultLike(details: (Contact, PaymentDetails, String, Option[String])) {
```

Comment:
can we make this a case class instead of a tuple, then we can give a helpful name (other than _3) to that String and Option[String]?

Reply:
Good shout. I will do this.